content/post/2019-05-23-keep-up-with-cran.Rmarkdown

```diff
@@ -0,0 +1,43 @@
+---
+title:  How to keep up with CRAN policies and processes?
+date: '2019-05-21'
+slug:  keep-up-with-cran
+tags:
+  - help
+  - CRAN
+---
+
+CRAN, the Comprehensive R Archive Network, changes its rules and workflow every so often: see for instance [the new encoding setting of one of its check flavors](/2019/04/25/r-devel-linux-x86-64-debian-clang/). As a package developer, you'd better keep up with CRAN policies and processes to be able to safely retain your package(s) on CRAN and to prepare your next submissions with as much information as possible. We at R-hub are also every interested in not missing updates, in particular of the check setups, in order for our services to serve package developers as well as possible. In this post, we'll therefore answer this important question: **How** to keep up with CRAN policies and processes?
+
+## Direct information sources
+
+There is no direct hot flow of information from CRAN but there are at least two direct sources of information.
+
+The first one is the [CRAN policy](https://cran.r-project.org/web/packages/policies.html) that has however no changelog. Thankfully, in 2013 Dirk Eddelbuettel [created and announced](http://dirk.eddelbuettel.com/blog/2013/10/23/) his creating a tool, the CRAN Repository Policy Watch, that works by mirroring the SVN repository of the policy. You can use the tool in two ways,
+
+* Browse or even [watch](https://help.github.com/en/articles/watching-and-unwatching-repositories) the [mirror GitHub repository](https://github.com/eddelbuettel/crp);
```

Comment:
Watching does not help you much, as you only get notified about issues PRs, not the commits. 

Should we mention the original SVN account? It is here: https://svn.r-project.org/R-dev-web/ 

And I also have a mirror of it here: https://github.com/gaborcsardi/R-dev-web/tree/master/trunk/CRAN/Policy

Reply:
> Watching does not help you much, as you only get notified about issues PRs, not the commits.

I'll add this caveat. But don't you read your GitHub timeline? I do and see commits there :wink:

I'll mention the two other links, thanks.